### PR TITLE
Feature/8.available students page

### PR DESCRIPTION
**Pull request by agencik007**
created students navigation

### CODE REVIEW

File: src/components/Dashboard/Students.style.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      border-bottom: 1px solid red;
+   }
+`
+
+export const Text = styled.p`
+   font-family: Catamaran, sans-serif;
+   color: ${appColors.fontColor};
+   background: transparent;
+   font-size: 18px;
+`
```
Comment (Fremen1990) [inline src/components/Dashboard/Students.style.tsx]: Please use extension for style in .ts file as the rest of components are

**Review by Fremen1990 — CHANGES_REQUESTED** [review]
Please use extension for style in .ts file as the rest of components are

File: src/components/Dashboard/Students.style.tsx (hunk 1 above)
Comment (agencik007) [reply src/components/Dashboard/Students.style.tsx]: i did a little refactor

**Review by Fremen1990 — APPROVED** [review]
👍

**Pull request merged**